cfgrammar/src/lib/yacc/grammar.rs

```diff
@@ -555,6 +555,26 @@ where
         }
     }
 
+    /// Returns the name of a given Symbol `sym`.
```

Comment:
I moved the function into the place where it's used. Rather than documenting why it panics, maybe we should just pretty print EOF as `"EOF"`?

Reply:
No, we can't give it any name, because that might clash with a user defined name for another token (and then confusion will ensue!).

Reply:
That is true.

Reply:
Anyways, now that I moved this into `prod_to_string`, this can't happen, as EOF can't be a symbol within a production, right? So I can add an `unreachable!()` for that case.

Reply:
Correct.